### DISCUSSION TITLE
polyhedral meshes

### DISCUSSION THREAD

**Discussion by Zcaic** [su2code/SU2#1944] [General]
Hello!
Does SU2 support polyhedral meshes?

**Comment by pcarruscag** [comment]
No, but in theory the good thing of being vertex based is that a mesh of prisms and tets becomes a dual mesh of polys.

**Reply by Zcaic** [reply]
Thanks a lot！ And it there any tutorials about DES or LES？

**Reply by pcarruscag** [reply]
I'm not aware of any. Unfortunately, the scale-resolving folks are not very active currently so I don't know anyone I can refer you to.
